README.md

```diff
@@ -1 +1,37 @@
-# school-library-oop
\ No newline at end of file
+# School Library OOP
```

Comment:
As highlighted by the previous reviewer, our documentation needs to be informative as it can be of great benefit to users. 
- [ ] In your README kindly add instructions on how to set up and use your app locally. 

- [ ] Also include the **Prerequisites** section (as contained in the link shared with you) describing what users need to have in order to run the app.